### PR TITLE
Fix for #120

### LINKED ISSUE #120: Bind a Service to a Lambda - could not trigger the function
**Description**

The example https://github.com/kyma-project/examples/tree/master/service-binding/lambda does not work. The step "Trigger the function" gives the result:
```
ERRO[0000]
FATA[0000] an error on the server ("Error: 'EOF'
Trying to reach: 'http://10.28.3.28:8080/'") has prevented the request from succeeding (get services redis-client:http-function-port)
```

**Expected result**

The step 7 contains a command which triggers the function.

**Actual result**

Could not trigger the function

**Steps to reproduce**

Follow the test scenario

### PR DESCRIPTION
**Pull request by joek**
**Description**

Changes proposed in this pull request:

- Changed call to validate function
- Expose function for redis client

**Related issue(s)**
Resolves #120

### CODE REVIEW

File: service-binding/lambda/deployment/lambda-function.yaml [hunk 1]
```diff
@@ -64,3 +64,16 @@ spec:
...
+---
+apiVersion: gateway.kyma-project.io/v1alpha2
+kind: Api
+metadata:
+  labels:
+    function: redis-client
+    example: service-binding-lambda
+  name: service-binding-lambda
+spec:
+  hostname: redis-client-test.kyma.local
```
Comment (a-thaler) [inline service-binding/lambda/deployment/lambda-function.yaml]: There is no need to specify the full domain, just put the sub-domain here to have it more portable
```suggestion
  hostname: redis-client-test
```

File: service-binding/lambda/README.md [hunk 2]
```diff
@@ -56,15 +56,15 @@ Apply a battery of `yaml` files to run the example.
 7. Trigger the function.
     The information and statistics about the Redis server appear in the logs of the function Pod.
     ```bash
-    kubeless function call redis-client -n $KYMA_EXAMPLE_NS
+     curl curl -ik https://redis-client.kyma.local
```
Comment (a-thaler) [inline service-binding/lambda/README.md]: The API defines redis-client-test as host
curl is doubled
It is fixed for minikube scenario... Maybe use a variable like $KYMA_DOMAIN

File: service-binding/lambda/deployment/lambda-function.yaml [hunk 3]
```diff
@@ -64,3 +64,16 @@ spec:
...
+---
+apiVersion: gateway.kyma-project.io/v1alpha2
+kind: Api
+metadata:
+  labels:
+    function: redis-client
+    example: service-binding-lambda
+  name: service-binding-lambda
+spec:
+  hostname: redis-client-test
```
Comment (a-thaler) [inline service-binding/lambda/deployment/lambda-function.yaml]: The curl command is calling redis-client.xxx, so I guess the hostname needs to be adjusted to it?

Comment (joek) [reply service-binding/lambda/deployment/lambda-function.yaml]: nope, it will get the domain auto assigned.

**Pull request merged**